docs/apis/commands.md

```diff
@@ -104,6 +120,26 @@ apis:
         split_by: \s+
 ```
 
+As stated before, in Windows, you can also override the default shell, `cmd`, with, for example, powershell.
+
+```yaml
+# Used to get list of windows services
+name: windowsServiceList
+apis:
+  - name: windowsServiceList
+    commands:
+      - run: service
+        shell: powershell
+        event_type: WindowsCommand
+        split: horizontal
+        set_header: [status,name,displayname]
+        row_start: 1
+        regex_match: true
+        split_by: (\w+)\s+(\w+)\s+(\w+)
+```
+
+In this example we are executing a command, `service`, using Powershell as the command shell.
```

Comment:
replace: `Powershell` with `PowerShell`

if the above update is accepted; this needs to be changed as well:

> In this example we are executing the command `Get-Service`, using PowerShell as the command shell and ensuring proper spacing in the output table with the `-Autosize` parameter

Reply:
I've changed all to lowercase since that was the most common.

Reply:
`PowerShell` is the proper case for the shell, we use lowercase in the Flex `shell` selection, but anywhere else it should be properly formatted IMO

Reply:
ok, makes sense